library/src/main/java/ss_n/common_ktx/Permissions.kt

```diff
@@ -1,8 +1,53 @@
 package ss_n.common_ktx
 
 import android.Manifest
+import androidx.annotation.RequiresApi
 
 enum class Permissions(val permissions: Array<String>) {
+    /**
+     * [Manifest.permission.READ_MEDIA_IMAGES]
+     */
+    @RequiresApi(33)
+    IMAGE_READ(
+        arrayOf(
+            Manifest.permission.READ_MEDIA_IMAGES,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/sasa-nori/common-ktx/blob/c1248a1cd50c5a79422e8061213f8caa979265fc/library/src/main/java/ss_n/common_ktx/Permissions.kt#L13"></span>Not a valid Kotlin file (expecting an argument) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>